packages/kbn-text-based-editor/src/text_based_languages_editor.styles.ts

```diff
@@ -22,16 +22,16 @@ export const textBasedLanguageEditorStyles = (
   isCodeEditorExpandedFocused: boolean,
   hasReference: boolean,
   editorIsInline: boolean,
-  historyIsOpen: boolean
+  historyIsOpen: boolean,
+  hideHeaderWhenExpanded: boolean
 ) => {
-  let position = isCompactFocused ? ('absolute' as 'absolute') : ('relative' as 'relative'); // cast string to type 'relative' | 'absolute'
-  if (isCodeEditorExpanded) {
-    position = 'relative' as 'relative';
```

Comment:
I cannot exactly remember the why behind this change.. I think it was no longer necessary/causing issues.

Reply:
You are right, it doesnt seem to be needed anymore.